test/test_container.py

```diff
@@ -52,3 +52,23 @@ def test_container_manifest_generates_sbom(tmp_path, build_container):
     sbom_json = json.loads(image_sbom_json_path.read_text())
     # smoke test that we have glibc in the json doc
     assert "glibc" in [s["name"] for s in sbom_json["packages"]], f"missing glibc in {sbom_json}"
+
+
+@pytest.mark.skipif(os.getuid() != 0, reason="needs root")
+def test_container_build_generates_manifest(tmp_path, build_container):
+    output_dir = tmp_path / "output"
+    output_dir.mkdir()
+    subprocess.check_call([
+        "podman", "run",
+        "--privileged",
+        "-v", f"{output_dir}:/output",
+        build_container,
+        "build",
```

Comment:
Ideally we would use the "build_fake_container" here that mocks osbuild (c.f. https://github.com/osbuild/image-builder-cli/pull/21/commits/13279a2562eb3b4b3ea0932c4ca0c322f3baa12a) as for this particular test we don't really need the full build. But its fine, we can tweak it after the other one is merged (or if you prefer you could cherry pick that commit and replace "build_container" with "build_fake_container" in the test and it should run a lot quicker :)

Reply:
I think we do; because if we don't then the actual error (that was there) isn't triggered? Perhaps if we mock the filesystem as well; but that'd be a followup.